File: src/main/webapp/jsLibrary/js/service/meetingService.js
app.service("meetingService", function ($http) {

    this.findMeetingListAll = function () {
        return $http.post("../meeting/findMeetingListAll");
    }
    this.findPage = function (currentPage, pageSize, meeting) {
        return $http.post("../meeting/getMeetingList?currentPage=" + currentPage + "&pageSize=" + pageSize, meeting);
    }
    this.findMeetingNoteByMeetingId = function (meetingNotes) {
        return $http.post("../meeting/getMeetingNoteList", meetingNotes);
    }
    this.findMeetingSheduleByMeetingId = function (meetingShedule) {
        return $http.post("../meeting/getMeetingSheduleList", meetingShedule);
    }
    this.addMeeting = function (meeting) {
        return $http.post("../meeting/addMeeting", meeting);
    }
    this.addMeetingNotes = function (meetingNotes) {
        return $http.post("../meeting/addMeetingNotes", meetingNotes);
    }
    this.addMeetingShedule = function (meetingShedule) {
        return $http.post("../meeting/addMeetingShedule", meetingShedule);
    }

})

File: src/main/java/com/meeting/m2/service/MeetingService.java
package com.meeting.m2.service;

import com.meeting.m2.pojo.Meeting;
import com.meeting.m2.pojo.MeetingNotes;
import com.meeting.m2.pojo.MeetingShedule;
import com.meeting.m2.util.DataGrid;
import org.springframework.web.multipart.MultipartFile;

public interface MeetingService {

    DataGrid getMeetingList(int currentPage, int pageSize, Meeting meeting);

    Meeting getMeetingByPrimaryKey(long id);

    void addMeeting(Meeting meeting) throws Exception;

    void updateMeeting(Meeting meeting) throws Exception;

    DataGrid getMeetingNoteList(int currentPage, int pageSize, MeetingNotes meetingNotes);

    DataGrid getMeetingSheduleList(int currentPage, int pageSize, MeetingShedule meetingShedule);

    void addMeetingNotes(MeetingNotes meetingNotes);

    void addMeetingShedule(MeetingShedule meetingShedule);

    void importMeetingShedule(long meetingId, String meetingShedulListName, MultipartFile file);
}
